lv_conf_template.h

```diff
@@ -34,6 +34,21 @@
  *Requires `LV_COLOR_DEPTH = 32` colors and the screen's `bg_opa` should be set to non LV_OPA_COVER value*/
 #define LV_COLOR_SCREEN_TRANSP 0
 
+/**
```

Comment:
I might sound odd, but from the user perspective, a so marginal config appearing at the top of the config file seems noisy.

What about this?
```c
/* Adjust color mix functions rounding. GPUs might calculate color mix (blending) differently.
 * 0: round down, 64: round up from x.75, 128: round up from half, 192: round up from x.25, 254: round up */
#if LV_COLOR_DEPTH == 32
#define LV_COLOR_MIX_ROUND_OFS (LV_COLOR_DEPTH == 32 ? 0 : 128)
#endif
```